scheduler/roles/slurm/tasks/dbd.yml

```diff
@@ -0,0 +1,105 @@
+# Copyright 2025 Dell Inc. or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+- name: Is slurm_db_host defined
+  ansible.builtin.set_fact:
+    slurm_db_host: "{{ groups['slurm_dbd'][0] }}"
+  when: (slurm_db_host is not defined) or (not slurm_db_host)
+
+- name: Is slurm_db_port defined
+  ansible.builtin.set_fact:
+    slurm_db_port: "{{ slurm_db_port_default }}"
+  when: (slurm_db_port is not defined) or (not slurm_db_port)
+
+- name: Is slurm_db_type defined
+  ansible.builtin.set_fact:
+    slurm_db_type: "{{ slurm_db_type_default }}"
```

Comment:
validtion should fail if slurm_db_type is not defined. we should not take default value

Reply:
mariadb is the default- this wa discussed

Reply:
cluster validation takes care of this. default is mariadb